test/gtest/common/mem_buffer.cc

```diff
@@ -269,6 +275,12 @@ void *mem_buffer::allocate(size_t size, ucs_memory_type_t mem_type, bool async)
 
 void mem_buffer::release(void *ptr, ucs_memory_type_t mem_type, bool async)
 {
+    if (async && !is_async_supported(mem_type)) {
+        UCS_TEST_MESSAGE << "asynchronous release for "
+                         << std::string(ucs_memory_type_names[mem_type])
+                         << " memory type is not supported";
+    }
```

Comment:
i guess this should not happen because otherwise we wouldnt be able to allocate? in this case we can just not release anything IMO..

Reply:
Changed to `UCS_TEST_ABORT`.